packages/atomic/src/components/commerce/atomic-commerce-interface/atomic-commerce-interface.new.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import {parameters} from '@coveo/atomic/storybookUtils/common-meta-parameters';
+import {renderComponent} from '@coveo/atomic/storybookUtils/render-component';
+import {getSampleCommerceEngineConfiguration} from '@coveo/headless/commerce';
+import type {Meta, StoryObj as Story} from '@storybook/web-components';
+
+async function initializeSearchInterface(canvasElement: HTMLElement) {
+  await customElements.whenDefined('atomic-commerce-interface');
+  const searchInterface = canvasElement.querySelector(
+    'atomic-commerce-interface'
+  );
+  await searchInterface!.initialize({
+    ...getSampleCommerceEngineConfiguration(),
+    ...{},
+  });
+}
+const meta: Meta = {
+  component: 'atomic-commerce-interface',
+  title: 'Atomic-Commerce/Interfaces/atomic-commerce-interface',
+  id: 'atomic-commerce-interface',
+  render: renderComponent,
+  parameters,
+  play: async (context) => {
+    await initializeSearchInterface(context.canvasElement);
+    const searchInterface = context.canvasElement.querySelector(
+      'atomic-commerce-interface'
+    );
+    await searchInterface!.executeFirstRequest();
+  },
+};
+
+export default meta;
+
+export const Default: Story = {
+  name: 'atomic-commerce-interface',
+};
+
+export const SearchBeforeInit: Story = {
+  tags: ['commerce', 'test'],
+  play: async (context) => {
+    const searchInterface = context.canvasElement.querySelector(
+      'atomic-commerce-interface'
+    );
+    await searchInterface!.executeFirstRequest();
+  },
+};
+
+export const WithProductList: Story = {
+  tags: ['commerce', 'test'],
+  argTypes: {
+    'attributes-language': {control: 'text', table: {category: 'attributes'}},
+  },
```

Comment:
I see you used the argTypes already, good 👍 
However:
1. You should define it in the meta to reuse it in all the stories
2. Here's the one you need to be able to keep the optionality of the param:
```ts
  argTypes: {
    'attributes-language': {
      name: 'language',
      type: 'string',
    },
  },
  ```